src/DIRAC/Core/DISET/ServiceReactor.py

```diff
@@ -43,6 +43,8 @@
 
 __RCSID__ = "$Id$"
 
+THROTTLE_SERVICE_SLEEP_SECONDS = 0.25
```

Comment:
Could you add a comment what the effect of this sleep is here?
```
#: Time during which the service does not accept new requests and handles those in the queue, if the backlog is too large
```
(Did I understand the effect of this sleep correctly?)

Reply:
As it is currently implemented it's slightly different but I'll let you read the comment to check if it's understandable.

Reply:
Yes I understand :+1: 
But please keep the `#:` so the comment shows up in the documentation page.

Reply:
Ah I wasn't aware of that convention. Fixed!